test.js

```diff
@@ -387,6 +387,33 @@ test('posList with srsDimension of 3', (t) => {
 	t.end()
 })
 
+test('posList with invalid srsDimension of "three"', (t) => {
+	const p = h('gml:Polygon', [
+		h('gml:exterior', [
+			h('gml:LinearRing', [
+				h('gml:posList', {srsDimension: 'three'})
+			])
+		])
+	])
+
+	t.throws(()=>(parse(p), /invalid srsDimension attribute value "three", expected a positive integer/))
```

Comment:
Didn't review properly. I assume this is not how you intended to call `t.throws(fn, expected)`. Right now, you're passing only *one* argument to `t.throws`, and the function would return a `RegExp` if it *wouldn't throw*.

If I do this however, the test currently fails:

```js
t.throws(() => parse(p), /invalid srsDimension attribute value "three", expected a positive integer/)
```

Reply:
Fixed it on `master`.